playwright.config.ts

```diff
@@ -0,0 +1,45 @@
+import { PlaywrightTestConfig, devices } from "@playwright/test";
+
+const opts = {
+  // launch headless on CI, in browser locally
+  headless: !!process.env.CI || !!process.env.PLAYWRIGHT_HEADLESS,
+  collectCoverage: false, // not possible in Next.js 12
```

Comment:
```suggestion
```
Not possible by default or use these fixtures: https://github.com/mxschmitt/playwright-test-coverage/